src/charts/donut/examples/donut-pct-chart.js

```diff
@@ -26,6 +26,15 @@
  * <li>.tooltipFn(d)   - user defined function to customize the tool tip (optional)
  * <li>.centerLabelFn  - user defined function to customize the text of the center label (optional)
  * <li>.onClickFn(d,i) - user defined function to handle when donut arc is clicked upon.
+ * <li>.label          - object containing properties for external label (optional)
+ *   <ul>
+ *       <li>.orientation - string with possible values: 'left', 'right' (optional)
```

Comment:
Good catch can update the comments/annotation. For our purposes, the optional default, is it simply doesn't exist, don't have to use it